tools/openms/.shed.yml

```diff
@@ -1,12 +1,22 @@
-name: openms
-owner: galaxyp
-description: OpenMS Suite for LC/MS data management and analyses
-homepage_url: https://www.openms.de/
-long_description: |
-  OpenMS is an open-source software C++ library for LC/MS data management and analyses. It offers an infrastructure for the rapid development of mass spectrometry related software.
-
-  https://www.openms.de/
-remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms
-type: unrestricted
-categories:
-- Proteomics
\ No newline at end of file
+name: openms
+owner: galaxyp
+description: OpenMS Suite for LC/MS data management and analyses
+homepage_url: https://www.openms.de/
+long_description: |
+  OpenMS is an open-source software C++ library for LC/MS data management and analyses. It offers an infrastructure for the rapid development of mass spectrometry related software.
+
+  https://www.openms.de/
+remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms
+type: unrestricted
+categories:
+- Proteomics
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
```

Comment:
Do Tool IDs have to be unique? I would agree in changing the package names, but adjusting the tool IDs seems like a pretty random intervention in the automatic wrapper generation.